.ahoy.yml

```diff
@@ -111,7 +111,12 @@ commands:
 
   install-site:
     usage: Install a site.
-    cmd: docker-compose exec -e DREVOPS_DRUPAL_INSTALL_OPERATIONS_SKIP=${DREVOPS_DRUPAL_INSTALL_OPERATIONS_SKIP:-} -e DREVOPS_DRUPAL_INSTALL_OVERRIDE_EXISTING_DB=${DREVOPS_DRUPAL_INSTALL_OVERRIDE_EXISTING_DB:-0} -T cli ./scripts/drevops/drupal-install-site.sh
+    cmd: |
+      docker-compose exec \
+      -e DREVOPS_DRUPAL_INSTALL_OPERATIONS_SKIP=${DREVOPS_DRUPAL_INSTALL_OPERATIONS_SKIP:-} \
+      -e DREVOPS_DRUPAL_INSTALL_OVERRIDE_EXISTING_DB=${DREVOPS_DRUPAL_INSTALL_OVERRIDE_EXISTING_DB:-0} \
+      -e SKIP_SUBTHEME_ACTIVATION=${SKIP_SUBTHEME_ACTIVATION:-0} \
```

Comment:
added a flag to skip subtheme activation during site install - this allows to have an alternative build where CivicTheme is used without a subtheme